pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native platform command line client.
+> More information: <https://native.electrode.io/reference/index-6>.
+
+- Create a new `ern` application (`MiniApp`):
+
+`ern create-miniapp {{appName}}`
```

Comment:
We use snake case for our arguments
```suggestion
`ern create-miniapp {{app_name}}`
```

Reply:
Thank you for the explanation, changed to snake case also renamed to be more verbose "application_name" per the comment below.